blog/2015/02/weekly-update-2015-02-06.html

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog
+root: ../../..
+author: Anelda van der Walt
+title: "Weekly Update: Jan 31 - Feb 06, 2015"
+date: 2015-02-06
+time: "12:30:00"
+category: ["Community"]
+---
+<!-- start excerpt -->
+<table class="table table-striped">
+  <tr>
+    <td>Highlights:</td>
+    <td>SWF gets 2 new affiliates (<a href="{{page.root}}/blog/2015/02/iplant-becomes-swc-partner.html">iPlant</a> and <a href="{{page.root}}/blog/2015/02/ucdavis-becomes-swc-affiliate.html">Lab for Data Intensive Biology at UC Davis</a>) and a new partner (<a href="{{page.root}}/blog/2015/02/uw-becomes-swc-partner.html">University of Washington</a>)</td>
+  </tr>
+  <tr>
+    <td>Conversations:</td>
+    <td>"..what matters most in teaching is the act itself - the verb.." - <a href="{{page.root}}/blog/2015/02/nouns-and-verbs.html">Book recommendations to support great teaching </a></td>
+  </tr>
+  <tr>
+    <td></td>
+    <td><a href="{{page.root}}/blog/2015/02/huang-scientific-coding-vs-software-engineering.html">Scientific coding and software engineering: what's the difference?</a> - Post by Daisie Huang - please add your comments</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td><a href="{{page.root}}/blog/2015/02/rewarding-software-sharing.html">Rewarding Software Sharing by Mapping Scientific Software</a> - Post by Chris Bogart - please share your ideas</td>
+  </tr>
+    <tr>
+    <td>Events:</td>
+    <td>SWC instructor training & Intro to Web Programming at <a href="{{page.root}}/blog/2015/02/workshops-in-march-at-lbl.html">Lawrence Berkeley Lab</a> - March 2015</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td><a href="{{page.root}}/blog/2015/02/ropensci-unhackathon.html">rOpenSci Unconference</a> - March 2015</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td><a href="{{page.root}}/blog/2015/02/ssi-collaborations-workshop.html">Software Sustainability Institute's Collaborations Workshop</a> - March 2015</td>
+  </tr>
+  <tr>
+    <td>Other News:</td>
+    <td><a href="{{page.root}}/blog/2015/02/welcome-our-newest-instructors.html">Eight new instructors joined SWC</a></td>
+  </tr>
+</table>
+<!-- end excerpt -->
+<p><strong>Details</strong></p>
+<ul>
+  <li>
+    This week the first meeting of the new SWF steering committee took place
```

Comment:
What about

~~~
This week the first meeting of the new SWF steering committee took place. Minutes will be publish soon.
~~~